src/home/home.tsx

```diff
@@ -1,12 +1,15 @@
 import {ApolloProvider} from '@apollo/client';
 import React from 'react';
+import {NavigationComponentProps} from 'react-native-navigation';
+import {AddUserButton} from '../components/fab';
 import {UsersList} from '../components/users-list';
 import {client} from '../utils/apollo';
 
-export const HomeScreen = () => {
+export const HomeScreen = (props: NavigationComponentProps) => {
   return (
     <ApolloProvider client={client}>
       <UsersList />
+      <AddUserButton componentId={props.componentId} />
```

Comment:
aqui não seria melhor passar um `onTap` para o `AddUserButton`, e usar o `props.componentId` na `HomeScreen` mesmo?
```suggestion
// 'dentro' da HomeScreen
const handleAddUserFABTap = () => {
  // usar o props.componentId
}
      <AddUserButton onTap={handleAddUserFABTap} />
```